op-node/rollup/derive/engine_controller.go

```diff
@@ -32,7 +34,14 @@ const (
 	syncStatusFinishedEL                // EL sync is done & we should be performing consolidation
 )
 
-var errNoFCUNeeded = errors.New("no FCU call was needed")
+var (
+	errNoFCUNeeded             = errors.New("no FCU call was needed")
+	ErrELSyncTriggerUnexpected = errors.New("forced head needed for startup")
+
+	maxFCURetryAttempts = 5
+	fcuRetryDelay       = 5 * time.Second
```

Comment:
5 seconds seems a bit too long for a retry.
I suggest using a shorter time, such as 1 second or 2 seconds.

Reply:
after testing with high tps scenario, geth data deletion duration can reach up to 16 seconds, with most operations completing within 10 seconds. Given FCU's 5-second timeout, this retry strategy has been implemented accordingly.